sickbeard/search.py

```diff
@@ -41,20 +38,22 @@
 from sickbeard import ui
 from sickbeard import failed_history
 from sickbeard import common
+
 from sickrage.helper.encoding import ek
 from sickrage.helper.exceptions import AuthException, ex
 from sickrage.providers.GenericProvider import GenericProvider
 from sickrage.helper.common import enabled_providers
 
+from socket import timeout as SocketTimeout
+
 
 def _downloadResult(result):
     """
-    Downloads a result to the appropriate black hole folder.
+    Download a result to the appropriate black hole folder.
```

Comment:
Yes, it's part of [PEP 257 -- Docstring Conventions](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).
Although not explicitly mentioned, first line of multi-line docstrings should be in imperative mood and end with a point.
As you can also see it from the example in PEP 257:
```
def complex(real=0.0, imag=0.0):
    """Form a complex number.

    Keyword arguments:
    real -- the real part (default 0.0)
    imag -- the imaginary part (default 0.0)
    """
    if imag == 0.0 and real == 0.0:
        return complex_zero
    ...
``` 
[Other examples](http://dolphm.com/pep257-good-python-docstrings-by-example/).

Personally, I prefer the 3rd person as well, but I think PEP should have priority over personal preferences.